packages/commonwealth/server.ts

```diff
@@ -275,6 +279,22 @@ async function main() {
   const dbValidationService: DatabaseValidationService =
     new DatabaseValidationService(models);
 
+  try {
+    app.use(
+      '/trpc',
+      createExpressMiddleware({ router: appRouter, createContext }),
+    );
+    app.use(
+      '/trpc',
+      createOpenApiExpressMiddleware({ router: appRouter, createContext }),
+    );
+
+    app.use('/docs', swaggerUi.serve);
+    app.get('/docs', swaggerUi.setup(openApiDocument));
```

Comment:
Should be exposed to developers as API documentation. Getting the swagger spec automatically generated is the first step, we still need to decide how to expose it (api management tools)

Reply:
Maybe a little bit unrelated to the overall scheme of things, but we have been talking about unifying our internal and external APIs. This either means:
1. We remove the external API's
2. We provide documentation for users to access the internal API's

I think that we have decided to move ahead with option 2 (not 100% on this). Anyway's, from a practical point of view, since we are open source, all our routes are publicly available (in the source code). So for this reason I don't see why we shouldn't expose clear documentation for users to leverage our internal API's.